### DISCUSSION TITLE
content navigation pane highlighting not working

### DISCUSSION THREAD

**Discussion by yeshodhan** [pradyunsg/furo#305] [Potential Issue] [answered]
content navigation highlights seems to be broken due to below issue:

```
furo.js:122 Uncaught TypeError: e.Gumshoe is not a constructor
    at c (furo.js:122)
    at HTMLDocument.<anonymous> (furo.js:152)
```

issue can be observed on furo docs website too:
![Screen Shot 2021-11-15 at 8 14 13 AM](https://user-images.githubusercontent.com/2785164/141816611-b095a962-796a-4c97-be3e-2d33b7816f99.png)

**Comment by pradyunsg** [comment, marked as answer]
Dammit. That's for the report. I'll take a look in a bit -- I likely goofed up the JS build pipeline and compression.

**Reply by pradyunsg** [reply]
It was actually much simpler. :)

https://github.com/pradyunsg/furo/commit/6a861ea29753e4d4cfdd7f853794bdcbf8c99691
